lib/scala/kg/src/main/scala/stores/kg/mem/MemKgStore.scala

```diff
@@ -18,11 +20,36 @@ class MemKgStore extends KgStore {
   private val luceneNodeLabelsField = lucene.create.field[String]("labels", fullTextSearchable = true)
   private var nodes: List[KgNode] = List()
   private var nodesById: Map[String, KgNode] = Map()
+  private var nodesPageRanks: Map[String, Double] = Map()
```

Comment:
Add an optional field to KgNode for the pageRank, so it can be requested from the UI, too.

You can use .copy to mutate a case class after the fact.

Reply:
Added `pageRank: Option[Double]` to KgNode